### PR TITLE
Add missing to workload

### PR DESCRIPTION
**Pull request by MajidAbuRmila**

### CODE REVIEW

**Review by enk21 — CHANGES_REQUESTED** [review]
I made a commit with a few updates. Kyle, Dan said you can describe the "Parallel" scaling policy in the workload rollout options. In my test workloads, I don't see the following in the status: "health_check", "current_replica_count", "resolved_images".  I am waiting for a response from Dan if these are still work in progress, unless you have a workload you can send me a link for.

File: docs/resources/workload.md [hunk 1]
```diff
@@ -175,13 +185,22 @@ Optional:
...
+- **port** (Number) gRPC Port.
+
 <a id="nestedblock--container--volume"></a>
 
 ### `container.volume`
 
 Required:
 
 - **uri** (String) URI of volume at cloud provider.
+- **recovery_policy** (String) Only applicable to persistent volumes, this determines what Control Plane will do when creating a new workload replica if a corresponding volume exists. Available Values: `retain`, `recycle`. Default: `retain`.
```
Comment (Cuppojoe) [inline docs/resources/workload.md:203]: We can leave recovery_policy, but please add a deprecated warning. This field is no longer used.

Comment (Cuppojoe) [reply docs/resources/workload.md:203]: @MajidAbuRmila

**Comment by Cuppojoe** [comment]
@enk21 The Parallel scaling policy will cause all pods affected by a scaling operation to be created or destroyed simultaneously. This does not affect update operations.

**Comment by Cuppojoe** [comment]
@enk21 

- health_check: I'm not sure what this one is
- current_replica_count: ditto
- resolved_images: this one only appears if you have enabled dynamic tags on the workload.

**Review by enk21 — APPROVED** [review]

**Pull request merged**